trex/mini2Dx-sample-trex-runner/core/src/main/java/com/mystudio/ChromeTrexGame/ChromeTrexGame.java

```diff
@@ -0,0 +1,144 @@
+package com.mystudio.ChromeTrexGame;
+
+import org.mini2Dx.core.Graphics;
+import org.mini2Dx.core.Mdx;
+import org.mini2Dx.core.collision.CollisionBox;
+import org.mini2Dx.core.font.GameFont;
+import org.mini2Dx.core.game.BasicGame;
+import org.mini2Dx.core.graphics.Texture;
+import org.mini2Dx.core.graphics.viewport.FitViewport;
+import org.mini2Dx.core.graphics.viewport.Viewport;
+import sun.rmi.runtime.Log;
+
+
+import static com.mystudio.ChromeTrexGame.Sounds.playDeathSound;
+
+
+public class ChromeTrexGame extends BasicGame {
+    public static final String GAME_IDENTIFIER = "com.mystudio.ChromeTrexGame";
+    private static final String BACKGROUND_TEXTURE_LOCATION = "backrgound.png";
+    //private static final String BOTTOM_TEXTURE_LOCATION = "ground.png";
+    public static float GRAVITY = 1.0f;
+
+    public static boolean inGame, jumping;
+    public static final float GAME_WIDTH = 500;
+    public static final float GAME_HEIGHT = 500;
+    private static float GAME_GRAVITY = 0.6f;
+
+
+    private boolean deadTrex;
+    public static int trexScore, highestScore;
+
+
+    InputHandler inputHandler;
+    Viewport fitViewport;
+    Trex trexTexture;
+    TrexData trexData;
+    Trex trex;
+    Obstacle obstacle;
+    Obstacle[] obstacles;
+    Obstacle obstacleTexture;
+    Sounds gameSounds;
+    //colisiones de el techo y el suelo del juego
+    CollisionBox[] colliderBottom, obstacleCollider;
+    Texture background;
+    //Texture ground;
+    UI ui;
```

Comment:
Unless these fields need to be accessed by other classes in the same package, change them to private.